lib/screens/v2/disruption_diagram/label.ex

```diff
@@ -109,4 +109,8 @@ defmodule Screens.V2.DisruptionDiagram.Label do
       "place-hsmnl" in end_stop_ids -> "place-hsmnl"
     end
   end
+
+  def get_end_label_id(_end_stop_ids, :mattapan, :trunk) do
+    "place-matt"
```

Comment:
Just checking: do we not need to return Ashmont in some cases here?

Reply:
No, since I always configured the Mattapan disruption diagram to face the same direction, the end station to label will always be the Mattapan stop.